app/views/layouts/application.html.slim

```diff
@@ -27,6 +27,10 @@ html lang="en" class="govuk-template"
     = govuk_header(service_name: t("service.name")) 
 
     div class="govuk-width-container"
+
+      - if content_for?(:back)
+        = yield :back
+
```

Comment:
Not a big deal, but it appears that we don't need the check: https://apidock.com/rails/ActionView/Helpers/CaptureHelper/content_for%3F